src/lib/responses-compat.ts

```
@@ -1193,70 +1405,649 @@ function parseResponsesSsePayloads(streamText: string): Array<Record<string, unk
+    name: "",
+    arguments: "",
+    server_label: "",
+    status: "in_progress",
+  }));
+
+  return asString(item["type"]) === "mcp_call"
+    ? item
+    : null;
+}
+
+function materializeResponsesEventPayloads(
```

ESLint

[error] Function 'materializeResponsesEventPayloads' has a complexity of 132. Maximum allowed is 50.

(complexity)